src/style.css

```diff
@@ -0,0 +1,20 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin: 0;
```

Comment:
`p` tags should also have no outer spacing. You may remove `margin: 0` from here and write it below like this:

```
h1,
p {
  margin: 0;
}
```

That will mean that both `h1` and `p` will have no outer spacings